airbyte-integrations/connectors/source-quickbooks-singer/build.gradle

```diff
@@ -1,21 +1,14 @@
 plugins {
     id 'airbyte-python'
     id 'airbyte-docker'
-    id 'airbyte-standard-source-test-file'
+    id 'airbyte-source-acceptance-test'
 }
 
 airbytePython {
     moduleDirectory 'source_quickbooks_singer'
 }
 
-airbyteStandardSourceTestFile {
-    specPath = "source_quickbooks_singer/spec.json"
-    configPath = "secrets/config.json"
-    configuredCatalogPath = "sample_files/configured_catalog.json"
-}
-
-
 dependencies {
-    implementation files(project(':airbyte-integrations:bases:base-standard-source-test-file').airbyteDocker.outputs)
+    implementation files(project(':airbyte-integrations:bases:source-acceptance-test').airbyteDocker.outputs)
     implementation files(project(':airbyte-integrations:bases:base-singer').airbyteDocker.outputs)
```

Comment:
don't need docker image for singer anymore, we are using cdk PYPI package, which already includes singer wrappers
```suggestion
```